.github/workflows/tests.yaml

```
@@ -0,0 +1,75 @@
+name: Tests and linting
+
+on:
+    push:
+    pull_request:
+    schedule:
+        - cron:  '0 3 * * *'
+
+jobs:
+    unit-tests:
+        runs-on: ubuntu-latest
+
+        strategy:
+            matrix:
+                php-version: ['7.3']
+                dependencies: ['']
+                include:
+                    - { php-version: '7.3', dependencies: '--prefer-lowest --prefer-stable' }
+
+        name: Unit tests - PHP ${{ matrix.dependencies }}
```

PHP version is missing from the name:

```suggestion
        name: Unit tests - PHP ${{ matrix.php-version }} ${{ matrix.dependencies }}
```

---

I omitted it intentionally, so the build `Unit test - PHP` could be marked as required and wont change with php updates. And since now and in the upcomming changes will always be just one version of php I think it's better that way.

---

Ok, but then you will add PHP 8, forget about this and have builds with duplicate names :-D

---

No I wont `add` php 8, I will change 7.4 to 8 - I don't want to support more major versions of php in one version of api-filter (thats why I omitted the php version in the first place)